api/provider/aws/service_read.go

```diff
@@ -53,9 +54,31 @@ func (s *ServiceProvider) Read(serviceID string) (*models.Service, error) {
 	var loadBalancerID string
 	if len(ecsService.LoadBalancers) != 0 {
 		loadBalancer := ecsService.LoadBalancers[0]
-		loadBalancerName := aws.StringValue(loadBalancer.LoadBalancerName)
-		fqLoadBalancerID := loadBalancerName
-		loadBalancerID = delLayer0Prefix(s.Config.Instance(), fqLoadBalancerID)
+
+		// classic load balancer
+		if loadBalancer.LoadBalancerName != nil {
+			fqLoadBalancerID := aws.StringValue(loadBalancer.LoadBalancerName)
+			loadBalancerID = delLayer0Prefix(s.Config.Instance(), fqLoadBalancerID)
+		}
+
+		// application load balancer
+		if loadBalancerID == "" && loadBalancer.TargetGroupArn != nil {
+			readTargetGroup(s.AWS.ALB, nil, loadBalancer.TargetGroupArn)
```

Comment:
does this return an error? if so, can't ignore it here.

Reply:
✅ this call isn't needed.